tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/AspNetCore5AsmRulesToggle.cs

```diff
@@ -0,0 +1,73 @@
+// <copyright file="AspNetCore5AsmRulesToggle.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.RcmModels.Asm;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.RemoteConfigurationManagement;
+using Datadog.Trace.RemoteConfigurationManagement.Protocol;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests.Rcm
+{
+    public class AspNetCore5AsmRulesToggle : RcmBase
+    {
+        private const string ASMProduct = "ASM";
+
+        public AspNetCore5AsmRulesToggle(ITestOutputHelper outputHelper)
+            : base(outputHelper, testName: nameof(AspNetCore5AsmRulesToggle))
+        {
+            SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "0");
+        }
+
+        // TODO adjust third parameter as the following PRs are merged:
+        // * https://github.com/DataDog/dd-trace-dotnet/pull/3120
+        // * https://github.com/DataDog/dd-trace-dotnet/pull/3171
```

Comment:
These are merged/closed now AFAICT, should this test be updated, or just the comments removed?

Reply:
Fixed in e7109b55593a74ad58f72d0c6cd1d0dde97492b6